src/Compilers/CSharp/Desktop/CommandLine/CommandLineParser.cs

```diff
@@ -587,7 +595,18 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                                 }
                                 else
```

Comment:
You can just click on the expander to left of the greyed out line to see a little bit more context.